### PR TITLE
[pfcwd] Add single asic unit tests for show commands

### PR DESCRIPTION
**Pull request by neethajohn**
<!--
Please make sure you've read and understood our contributing guidelines:
https://github.com/Azure/SONiC/blob/gh-pages/CONTRIBUTING.md

** Make sure all your commits include a signature generated with `git commit -s` **

If this is a bug fix, make sure your description includes "closes #xxxx",
"fixes #xxxx" or "resolves #xxxx" so that GitHub automatically closes the related
issue when the PR is merged.

If you are adding/modifying/removing any command or utility script, please also
make sure to add/modify/remove any unit tests from the tests
directory as appropriate.

If you are modifying or removing an existing 'show', 'config' or 'sonic-clear'
subcommand, or you are adding a new subcommand, please make sure you also
update the Command Line Reference Guide (doc/Command-Reference.md) to reflect
your changes.

Please provide the following information:
-->

**- What I did**
Added single asic unit tests for pfcwd show commands

**- How to verify it**
```
platform linux2 -- Python 2.7.16, pytest-3.10.1, py-1.7.0, pluggy-0.8.0 -- /usr/bin/python2
cachedir: .pytest_cache
rootdir: /sonic/src/sonic-utilities/tests, inifile: pytest.ini
plugins: cov-2.6.0
collected 8 items                                                                                                                                                                                                                       

tests/pfcwd_test.py::TestPfcwd::test_pfcwd_show_config PASSED                                                                                                                                                                     [ 12%]
tests/pfcwd_test.py::TestPfcwd::test_pfcwd_show_config_single_port PASSED                                                                                                                                                         [ 25%]
tests/pfcwd_test.py::TestPfcwd::test_pfcwd_show_config_multi_port PASSED                                                                                                                                                          [ 37%]
tests/pfcwd_test.py::TestPfcwd::test_pfcwd_show_config_invalid_port PASSED                                                                                                                                                        [ 50%]
tests/pfcwd_test.py::TestPfcwd::test_pfcwd_show_stats PASSED                                                                                                                                                                      [ 62%]
tests/pfcwd_test.py::TestPfcwd::test_pfcwd_show_stats_single_queue PASSED                                                                                                                                                         [ 75%]
tests/pfcwd_test.py::TestPfcwd::test_pfcwd_show_stats_multi_queue PASSED                                                                                                                                                          [ 87%]
tests/pfcwd_test.py::TestPfcwd::test_pfcwd_show_stats_invalid_queue PASSED                                                                                                                                                        [100%]
```

Config commands pending on the multi asic db mock

### CODE REVIEW

File: tests/pfcwd_test.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+import os
+import sys
+from click.testing import CliRunner
+from utilities_common.db import Db
+from pfcwd_input.pfcwd_test_vectors import testData
```
Comment (jleveque) [inline tests/pfcwd_test.py:5]: Add whitespace to separate the imports per [PEP8 standards](https://www.python.org/dev/peps/pep-0008/#imports):

> Imports should be grouped in the following order:

> Standard library imports.
> Related third party imports.
> Local application/library specific imports.
> You should put a blank line between each group of imports.

And group alphabetically in each group (these are already in alphabetical order). This should read:

```python
import os
import sys

from click.testing import CliRunner
from utilities_common.db import Db

from pfcwd_input.pfcwd_test_vectors import testData
```

**Comment by smaheshm** [comment]
Looks good! Waiting for unit test result.

**Comment by smaheshm** [comment]
retest this please

**Comment by abdosi** [comment]
@neethajohn Please create PR for 201911

**Comment by smaheshm** [comment]
@neethajohn @abdosi Will create one PR for both #1080 and #1085

**Comment by smaheshm** [comment]
> @neethajohn Please create PR for 201911

#1102

**Pull request merged**